pkg/preflight/preflight.go

```diff
@@ -45,7 +45,7 @@ const (
 	SpecDataKey = "preflight-spec"
 )
 
-func Run(appID string, appSlug string, sequence int64, isAirgap bool, archiveDir string) error {
+func Run(appID string, appSlug string, sequence int64, isAirgap bool, ignoreNonStrict bool, archiveDir string) error {
```

Comment:
Added this variable to the `Run` function that can be populated by the caller based on if we're skipping preflights and if there are strict preflights.